README.md

```diff
@@ -78,8 +79,11 @@ Other Style Guides
     console.log(foo, bar); // => 1, 9
     ```
 
-  <a name="types--complex"></a><a name="1.2"></a>
-  - [1.2](#types--complex)  **Complex**: When you access a complex type you work on a reference to its value.
+  <a name="types--symbols"></a><a name="1.2"></a>
+  - [1.2](#types--symbols)  **Symbols**: Symbols cannot be faithfully polyfilled, so they should not be used when targeting browsers/environments that don't support them natively.
+
+  <a name="types--complex"></a><a name="1.3"></a>
```

Comment:
The legacy numbered links need to stay; iow, the "complex" section needs to forever be linked under `1.2`.